test/test_helpers/partners.rb

```diff
@@ -12,8 +12,12 @@ def teardown
     end
 
     def stub_partner_req(query, variables: {}, status: 200, resp: {})
+      filepaths = Dir[File.join(ShopifyCli::ROOT, 'lib', '**', 'graphql', "#{query}.graphql")]
+      if filepaths.count > 1
+        raise "Multiple queries in the codebase with filename #{query}.graphql, please rename your query."
+      end
```

Comment:
I added this because we are just taking the first path for tests so if there are multiple queries named the same, it would cause tests to be flaky. I chose to raise here instead of making the tests more complicated.

Reply:
Adding tests for test heleprs? :suspect:

Reply:
🤦, didn't realize this was a test helper. Moving on...